db/internal_stats.cc

```diff
@@ -851,6 +869,45 @@ bool InternalStats::HandleLiveBlobFileGarbageSize(uint64_t* value,
   return true;
 }
 
+bool InternalStats::GetBlobCacheForStats(Cache** blob_cache) {
+  assert(blob_cache != nullptr);
+  *blob_cache = cfd_->ioptions()->blob_cache.get();
+  return *blob_cache != nullptr;
+}
```

Comment:
We could consider changing this to the logically equivalent but simpler/cleaner to `Cache* InternalStats::GetBlobCacheForStats() { return cfd_->ioptions()->blob_cache.get(); }`

Reply:
Done, also changed `GetBlockCacheForStats`